docs/source/CONTRIBUTING.md

```diff
@@ -1,43 +1,57 @@
+- Create a [pull request](https://github.com/intel/neural-compressor/pulls).
 
 ## Pull Request Checklist
 
 Before sending your pull requests, follow the information below:
 
 - Changes are consistent with the Python [Coding Style](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
-- Use pylint to check your Python code.
-- Use flake8 and autopep8 to make Python code clean.
 - Add unit tests in [Unit Tests](https://github.com/intel/neural-compressor/tree/master/test) to cover the code you would like to contribute.
-- Run [Unit Tests](https://github.com/intel/neural-compressor/tree/master/test).
-- Intel® Neural Compressor has adopted the [Developer Certificate of Origin](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin), you must agree to the terms of Developer Certificate of Origin by signing off each of your commits.`Signed-off-by: Random J Developer <random@developer.example.org>`
+- Intel® Neural Compressor has adopted the [Developer Certificate of Origin](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin), you must agree to the terms of Developer Certificate of Origin by signing off each of your commits with `-s`, e.g. `git commit -s -m 'This is my commit message'`.
 
 ## Pull Request Template
 
-**Type of Change**  
-feature or bug fix or documentation or validation or others
-API changed or not
+See [PR template](/.github/pull_request_template.md)
+
+## Pull Request Acceptance Criteria
+- At least two approvals from reviewers
 
-**Description**  
-detail description  
-JIRA ticket: xxx
+- All detected status checks pass
 
-**Expected Behavior & Potential Risk**  
-the expected behavior that triggered by this PR
+- All conversations solved
 
-**How has this PR been tested?**  
-how to reproduce the test (including hardware information)
+- Third-party dependency license compatible
 
-**Dependency Change?**  
-any library dependency introduced or removed
+## Pull Request Status Checks Overview
+Intel® Neural Compressor use [Azure DevOps](https://learn.microsoft.com/en-us/azure/devops/pipelines/?view=azure-devops) for CI test.
+And generally use [Azure Cloud Instance](https://azure.microsoft.com/en-us/pricing/purchase-options/pay-as-you-go) to deploy pipelines, e.g. Standard E16s v5.
+|     Test Name                 |     Test Scope                                |     Test Pass Criteria    |
+|-------------------------------|-----------------------------------------------|---------------------------|
+|     Code Scan                 |     Pylint/Bandit/CopyRight/DocStyle/SpellCheck       |     PASS          |
+|     [DCO](https://github.com/apps/dco/)     |     Use `git commit -s` to sign off     |     PASS          |
+|     Unit Test                 |     Pytest scripts under [test](/test)                |      PASS (No failure, No core dump, No segmentation fault, No coverage drop)      |
```

Comment:
Do we have test coverage target or current coverage number available to contributor if he/she want to check if there is a drop?  or the UT result will report current coverage and new coverage with PR?

Reply:
The UT test will report a comparison table between this PR and master, so user could check the coverage and also able to see with file has regression. It's hard for user to get a full coverage data of their PR locally, it require several test ENV and combine all the test results to get the full coverage.

Reply:
Like this one: 
![image](https://github.com/intel/neural-compressor/assets/51692656/edf464e7-9aec-4f8f-b500-435910796787)

Reply:
I just experienced similar UT coverage result in my PR to ITREX, thanks for the clarification